msteams-platform/concepts/deploy-and-publish/appsource/submit-your-app.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Submit your app to Partner Center
+description: Learn how to create a Partner Center account and submit your app for store validation.
+author: heath-hamilton
+ms.author: lajanuar
+ms.topic: how-to
+---
+# Submit your Microsoft Teams app to Partner Center
```

Comment:
Wondering if we should just be linking to the existing M365 stores submission docs? https://docs.microsoft.com/office/dev/store/add-in-submission-guide